README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+# React Redux Capstone Project (Bank Holidays in United Kingdom)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> In this project, we will be working with the real live data from the UK Govt API. We are going to build a web application that tells us the holidays in UK calender. Bank holidays in England and Wales, Scotland and Northern Ireland
 
```

Comment:
- [ ] Please as it is a requirement for this project, please kindly deploy your application using `Heroku`, `Netlify` or `GitHub Pages` (or any platform of your choice).
- [ ] Please add a link to your  deployed project in this `README` file.